src/GraphQl/Resolver/Stage/SecurityPostDenormalizeStage.php

```diff
@@ -52,7 +57,7 @@ public function __invoke(string $resourceClass, string $operationName, array $co
             }
         }
 
-        if (null === $isGranted || $this->resourceAccessChecker->isGranted($resourceClass, (string) $isGranted, $context['extra_variables'])) {
+        if (null === $isGranted || null === $this->resourceAccessChecker || $this->resourceAccessChecker->isGranted($resourceClass, (string) $isGranted, $context['extra_variables'])) {
```

Comment:
This change is unnecessary. It's not possible at this point that `null !== $isGranted && null === $this->resourceAccessChecker` would evaluate to `true` (exactly what we've already checked above). (I'd also suggest changing the above condition to this form, as it reads better.)

Reply:
Ups! That was my bad I miss that check in there, fixing it now.